docs/machine-learning/tutorials/sentiment-analysis.md

```diff
@@ -95,115 +91,109 @@ The input dataset class, `SentimentData`, has a `string` for the comment (`Senti
+    {
 
-```csharp
-public static ITransformer BuildAndTrainModel(MLContext mlContext, IDataView splitTrainSet)
-{
+    }
+    ```
 
-}
-```
-
-## Extract and transform the data
-
-Call `FeaturizeText` as the next line of code:
-
-[!code-csharp[FeaturizeText](~/samples/machine-learning/tutorials/SentimentAnalysis/Program.cs#FeaturizeText "Featurize the text")]
-
-The `FeaturizeText()` method in the previous code converts the text column (`SentimentText`) into a numeric key type `Features` column used by the machine learning algorithm and adds it as a new dataset column:
-
-|SentimentText                         |Sentiment |Features              |
-|--------------------------------------|----------|----------------------|
-|Waitress was a little slow in service.|    0     |[0.76, 0.65, 0.44, …] |
-|Crust is not good.                    |    0     |[0.98, 0.43, 0.54, …] |
-|Wow... Loved this place.              |    1     |[0.35, 0.73, 0.46, …] |
-|Service was very prompt.              |    1     |[0.39, 0, 0.75, …]    |
+### Extract and transform the data
+<!---The transform and featurization parts of this section seem like they'd be under "prepare data" above. A quirk of the ML.NET process? -->
 
-## Add a learning algorithm
+1. Call `FeaturizeText` as the next line of code:
 
-### About the classification task
+    [!code-csharp[FeaturizeText](~/samples/machine-learning/tutorials/SentimentAnalysis/Program.cs#FeaturizeText "Featurize the text")]
 
-"Is it A or B?"
+    The `FeaturizeText()` method in the previous code converts the text column (`SentimentText`) into a numeric key type `Features` column used by the machine learning algorithm and adds it as a new dataset column:
 
-![classification machine learning algorithm](./media/sentiment-analysis/classification.png)
+    |SentimentText                         |Sentiment |Features              |
+    |--------------------------------------|----------|----------------------|
+    |Waitress was a little slow in service.|    0     |[0.76, 0.65, 0.44, …] |
+    |Crust is not good.                    |    0     |[0.98, 0.43, 0.54, …] |
+    |Wow... Loved this place.              |    1     |[0.35, 0.73, 0.46, …] |
+    |Service was very prompt.              |    1     |[0.39, 0, 0.75, …]    |
 
-Classification is a machine learning algorithm that uses data to **determine** the category, type, or class of an item or row of data and is frequently one of the following types:
+### Add a learning algorithm
 
-* Binary: either A or B.
-* Multiclass: multiple categories that can be predicted by using a single model.
-
-Because the website comments need to be classified as either positive or negative, you use the Binary Classification task.
+This app uses a classification algorithm that categorizes items or rows of data. The app categorizes website comments as either positive or negative, so use the binary classification task.
```

Comment:
would classify be a better choice than categorize?

Reply:
It's best not to use essentially the same word to define a word.